packages/@n8n/task-runner/src/runner-messages.ts

```diff
@@ -0,0 +1,204 @@
+import type { INodeTypeBaseDescription } from 'n8n-workflow';
+
+import type { RPC_ALLOW_LIST, TaskDataRequestParams, TaskResultData } from './runner-types';
+
+export namespace N8nMessage {
```

Comment:
I find `N8n-` vague as "n8n" can also be misinterpreted to be the requester. Since we're touching the types maybe we can be more explicit?

```suggestion
export namespace BrokerMessage {
```